manifests/elasticsearch.pp

```diff
@@ -17,7 +17,7 @@
   $elasticsearch_path_logs = '/var/log/elasticsearch',
 
 
-  $elasticsearch_ip = '<YOUR_ELASTICSEARCH_IP>',
+  $elasticsearch_ip = '0.0.0.0',
```

Comment:
I'd change this by `localhost` because listening on all interfaces is not required in single-host architecture .

Reply:
Hi @manuasir 

Change has been applied in https://github.com/wazuh/wazuh-puppet/pull/178/commits/225280231ef3909094e6b0b838885f501ac3ed25.

Kr,

Rshad